packages/exportable-manifest/src/index.ts

```diff
@@ -72,7 +72,16 @@ async function makePublishDependency (depName: string, depSpec: string, dir: str
   if (!depSpec.startsWith('workspace:')) {
     return depSpec
   }
-  if (depSpec === 'workspace:*' || depSpec.endsWith('@*')) {
+  if (
+    depSpec.endsWith('@*') || depSpec === 'workspace:*' ||
+    depSpec === 'workspace:^' || depSpec === 'workspace:~'
+  ) {
+    const rangeToken = depSpec === 'workspace:^'
```

Comment:
doesn't make sense to init the variable here as it is only used if the function doesn't exit at line 87.

Should be moved to line 93